### PR TITLE
Update PositionedComponentOffsets.cs

### PR DESCRIPTION
**Pull request by wlastas**
please help correct Reaction - its wrong now:)(

### CODE REVIEW

File: GameOffsets/PositionedComponentOffsets.cs [hunk 1]
```diff
@@ -7,16 +7,16 @@ namespace GameOffsets
...
-        [FieldOffset(0x1EC)] public int GridY;
+        [FieldOffset(0x268)] public int GridX; // 3.16.2b
+        [FieldOffset(0x26C)] public int GridY; // 3.16.2b
         [FieldOffset(0x1F0)] public float Rotation;
         [FieldOffset(0x208)] public float Scale;
         [FieldOffset(0x20C)] public int Size;
-        [FieldOffset(0x218)] public Vector2 WorldPosition;
-        [FieldOffset(0x218)] public float WorldX;
-        [FieldOffset(0x21C)] public float WorldY;
+        [FieldOffset(0x22C)] public Vector2 WorldPosition; //3.16.2b
```
Comment (justagithubdude) [inline GameOffsets/PositionedComponentOffsets.cs:18]: Are you sure this is correct? there's a bunch of "click position" offsets that look very alike if you don't watch a long movement or get the offsets from code

Comment (justagithubdude) [reply GameOffsets/PositionedComponentOffsets.cs:18]: In my PR, this is the previous position, which fits the previous layout (at least the field order), but I didn't check the game's code

Comment (wlastas) [reply GameOffsets/PositionedComponentOffsets.cs:18]: 0x1D9 its correct Reaction
Tested on my golrms and herald of purity mobs:))

Comment (wlastas) [reply GameOffsets/PositionedComponentOffsets.cs:18]: ok i see its dont need more;)) thanks

Comment (wlastas) [reply GameOffsets/PositionedComponentOffsets.cs:18]: > In my PR, this is the previous position, which fits the previous layout (at least the field order), but I didn't check the game's code
i also found the previous position on the grid
 [FieldOffset(0x1DC)] public StdTuple2D<int> PrevGpos;

**Pull request closed without merging**